browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -27,44 +36,89 @@ int OnBeforeURLRequest_StaticRedirectWork(
     URLPattern(URLPattern::SCHEME_HTTPS, "https://static.brave.com/*"),
-    // We do allow redirects to the Google update server for extensions we don't support
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://update.googleapis.com/service/update2"),
+    // We do allow redirects to the Google update server for extensions we don't
+    // support
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://update.googleapis.com/service/update2"),
 
     // Rewards URLs
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://ledger.mercury.basicattentiontoken.org/*"),
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://balance.mercury.basicattentiontoken.org/*"),
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://publishers.basicattentiontoken.org/*"),
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://publishers-distro.basicattentiontoken.org/*"),
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://ledger-staging.mercury.basicattentiontoken.org/*"),
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://balance-staging.mercury.basicattentiontoken.org/*"),
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://publishers-staging.basicattentiontoken.org/*"),
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://publishers-staging-distro.basicattentiontoken.org/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://ledger.mercury.basicattentiontoken.org/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://balance.mercury.basicattentiontoken.org/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://publishers.basicattentiontoken.org/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://publishers-distro.basicattentiontoken.org/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://ledger-staging.mercury.basicattentiontoken.org/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://balance-staging.mercury.basicattentiontoken.org/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://publishers-staging.basicattentiontoken.org/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://publishers-staging-distro.basicattentiontoken.org/*"),
 
     // Safe browsing
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://safebrowsing.brave.com/v4/*"),
-    URLPattern(URLPattern::SCHEME_HTTPS, "https://ssl.gstatic.com/safebrowsing/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://safebrowsing.brave.com/v4/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS,
+      "https://ssl.gstatic.com/safebrowsing/*"),
+
+    // CRLSets
+    URLPattern(URLPattern::SCHEME_HTTPS, "https://crlsets1.brave.com/*"),
+    URLPattern(URLPattern::SCHEME_HTTPS, "https://crlsets2.brave.com/*"),
 
-    // Will be removed when https://github.com/brave/brave-browser/issues/663 is fixed
+    URLPattern(URLPattern::SCHEME_HTTPS, "https://crxdownload.brave.com/*"),
+
+    // Will be removed when https://github.com/brave/brave-browser/issues/663
+    // is fixed
     URLPattern(URLPattern::SCHEME_HTTPS, "https://www.gstatic.com/*"),
```

Comment:
can we remove this line as https://github.com/brave/brave-browser/issues/663 was fixed?